src/goodHealthApp/models.py

```diff
@@ -1,16 +1,40 @@
 from django.contrib.auth.models import User
 from django.db import models
+from django.conf import settings
 from django.utils.timezone import now
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+
+
+def get_upload_image_dir(instance, filename):
+    return "uploads/{mimeType}/{userid}/{file}".format(mimeType="images", userid=instance.user.username, file=filename)
 
 
 class UserProfile(models.Model):
-    """
-    UserProfile Model.
-    """
     user = models.OneToOneField(User, on_delete=models.CASCADE)
+    first_name = models.CharField(max_length=100)
+    last_name = models.CharField(max_length=100)
+    email = models.CharField(max_length=100)
+    password = models.CharField(max_length=100)
```

Comment:
These should be removed. This model has a one-to-one field with the default django user model with these existing fields.